src/commands/available.ts

```diff
@@ -0,0 +1,104 @@
+import cli from "cli-ux";
+import * as chalk from "chalk";
+import * as execa from "execa";
+
+import TartCommand from "../TartCommand";
+
+const tableFlags = cli.table.flags();
+
+export default class Available extends TartCommand {
+  static description = "lists available dumps";
+
+  static examples = [`$ tart list`];
+
+  static flags = {
+    ...TartCommand.flags,
+    ...tableFlags,
+    sort: { ...tableFlags.sort, default: "tag" },
+  };
+
+  async run() {
+    const { flags } = this.parse(Available);
+
+    const localTags = await this.getTags(".");
+    const remoteTags = await this.getTags(this.localConfig.repository || "");
+
+    // console.info("local tags: ", localTags);
+    // console.info("remote tags: ", remoteTags);
+
+    let results: { tag: string; sha: string; status?: string }[] = [];
+    let remaining: { tag: string; sha: string; status?: string }[] = [
+      ...remoteTags,
+    ];
+
+    localTags.forEach((ltag) => {
```

Comment:
I think it would be really helpful to have a comment, or pull this out into a function to describe what this is doing. Tricky thing to name though lol

What do you think of this?
```
processTagStatuses(localTags, remoteTags)
```
and all the code from lines 29 to 64 can go in there so at this level it's easy to know that it takes the two sets of tags and spits out something we send to `cli.table`?

Reply:
Added some comments. Thought it would be better then to pull it out into a separate method.